### PR TITLE
Release candidate for 0.7.0

### PR DESCRIPTION
**Pull request by apdavison**
Please comment on:

1) whether you think the current code base is ok to release as 0.7.0 (no showstopper bugs)

2) the release notes.

### CODE REVIEW

File: doc/installation.txt [hunk 1]
```diff
@@ -53,9 +53,22 @@ Since Bazaar is mostly written in Python, just installing the main Bazaar
...
+elsewhere (e.g. to `~/.bash_completion.d/`)
+is recommended.
+
+
+.. _python.org: https://www.python.org
 .. _Django: https://www.djangoproject.com
-.. _Mercurial: http://mercurial.selenic.com/
+.. _Mercurial: https://mercurial.selenic.com/
 .. _Subversion: http://subversion.tigris.org/
 .. _Git: http://git-scm.com/
```
Comment (timtroendle) [inline doc/installation.txt]: supports `https` as well...

**Comment by timtroendle** [comment]
Before this is going to be released there should be a decision whether to support Python 3.3 or not. Currently that is not consistent. In case Python 3.3. shall be supported, Travis should build and test 3.3 as well. In case Python 3.3 shall not be supported, there are a few changes in the documentation necessary.

I'd be totally fine not supporting it. I don't know of any reasons why one wouldn't upgrade -- but maybe there are.

**Comment by apdavison** [comment]
ok, let's not claim Python 3.3 support. If someone asks for it in future, we can probably add support quite easily.

File: doc/installation.txt (hunk 1 above)
Comment (apdavison) [inline doc/installation.txt]: The issue is not whether it supports `https`, but whether `http` redirects to `https`. The Mercurial website does this, the Git website does not.

(these changes came from running `make linkcheck` in the `doc` directory, and trying to eliminate unnecessary redirects).

Comment (timtroendle) [inline doc/installation.txt]: Understood. I just wondered... ;-)

**Comment by maxalbert** [comment]
A while back I tried the (then-latest) dev version on a Sumatra project which contained many simulation runs (a few hundred at least), and I noticed that the web interface had gotten *extremely* slow compared to before - I had to wait several minutes for anything to be displayed every time I returned to the dashboard view. This was shortly after the work on the data-centric view was merged, so my assumption was that it had something to do with it.

Unfortunately, I didn't have time to investigate this in more detail and can't even run Sumatra on the project right now because of a version incompatibility (which I won't have time to investigate for at least the next few weeks), so I'm afraid I can't provide any hard data for the time being. I just wanted to leave a note here to ask if anybody has tried Sumatra with large-ish projects recently and can either confirm my observation or dispel my concerns.

**Comment by apdavison** [comment]
I've tested the current version with a project containing about 500 records, and observed no slow-down in the web interface.

(however "smt list" was very slow, fixed by 2215893).

**Comment by maxalbert** [comment]
Ok, great. Thanks for checking!

**Pull request merged**